src/System.Text.Json/tests/Serialization/CustomConverterTests.Polymorphic.cs

```diff
@@ -147,13 +147,74 @@ public static void PersonConverterPolymorphic()
         }
 
         [Fact]
-        public static void NullPersonConverterPolymorphic()
+        public static void NullPersonConverterPolymorphicTypeDiscriminator()
         {
             var options = new JsonSerializerOptions();
-            options.Converters.Add(new PersonConverter());
+            options.Converters.Add(new PersonConverterWithTypeDiscriminator());
 
             Person person = JsonSerializer.Deserialize<Person>("null");
             Assert.Null(person);
         }
+
+        // A converter that can serialize an abstract Person type.
+        private class PersonPolymorphicSerializerConverter : JsonConverter<Person>
+        {
+            public override Person Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+            {
+                throw new NotSupportedException($"Deserializing not supported. Type={typeToConvert}.");
```

Comment:
>  (serialization only, not deserialization)

Why? Was this meant to showcase why the `Type` parameter is important in `Read(...)`?